### DISCUSSION TITLE
关于参考文献英文人名是否需要 capitalize 的问题和个人简历的问题

### DISCUSSION THREAD

**Discussion by liziwl** [tuna/thuthesis#702] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/23000702/137508025-f6f4d85a-1271-4192-b085-f59e7e80852b.png)

注意到 GBT7714-2015 规范需要英文人名大写，不太清为什么模版没有大写。

另外在个人简历的部分的部分条目能否使用 bib 的信息（可能需要另外弄个文件）进行输出，目前是需要直接 `\item` 手动填写

![image](https://user-images.githubusercontent.com/23000702/137507691-571583f2-6079-4abd-94cc-8f560922f4ed.png)

**Comment by zepinglee** [comment, marked as answer]
> 注意到 GBT7714-2015 规范需要英文人名大写，不太清为什么模版没有大写。

研究生院要求首字母大写，相当于在国标的基础上进行额外的规定，应当优先执行学校的格式。
 
> 另外在个人简历的部分的部分条目能否使用 bib 的信息（可能需要另外弄个文件）进行输出，目前是需要直接 `\item` 手动填写

我感觉这个功能用类似于本科生的 `survey` 中独立参考文献的方法应该能够实现，但是要控制“论文”和“专利”两部分的编号，总的来说比较复杂。

目前可行办法是，另外新建一个文件进行编译，再将 `.bbl` 的内容拷贝过来。

**Reply by liziwl** [reply]
多谢回复

**Reply by liziwl** [reply]
bibunits 的实现：https://github.com/liziwl/sustech-master-thesis/commit/e8c88669f23aa89592ee96fd9c41961c0bb9eb8a

主要是防止同学不要写错参考文献格式，计算编号的起始问题比较小。

**Comment by hushidong** [comment]
你如果使用biblatex的话，可以使用biblatex的refsection来实现第二个要求。

**Reply by liziwl** [reply]
我测试后觉得 refsection不太好弄编号，会和前面正文的参考文献编号耦合。我需要在个人简历部分重新编号且连续编号。最后是用 bibunits实现的，自行指定起始文献编号。

**Reply by hushidong** [reply]
之所以会耦合，那是你没有把两个部分区分清楚。refsection和bibunits一样的，不同的部分要放到不同的refsection里面的。

当然习惯bibtex的使用bibunits来实现也挺好的。
